compute/cloud-client/src/test/java/compute/FirewallIT.java

```diff
@@ -123,10 +123,9 @@ public void testListFirewallRules()
     try {
       compute.ListFirewallRules.listFirewallRules(PROJECT_ID);
       assertThat(stdOut.toString()).contains(FIREWALL_RULE_CREATE);
-    } catch (NotFoundException e) {
-      System.out.println("Rule already deleted! ");
-    } catch (InvalidArgumentException | NullPointerException e) {
-      System.out.println("Rule is not ready (probably being deleted).");
+    } catch (AssertionError e) {
```

Comment:
This test will never fail. I think we need to be more specific on the Assertions we are catching.

Reply:
Good point. I've modified it to a simple condition check and threw a specific exception.